src/utils/credentials.ts

```diff
@@ -86,9 +86,9 @@ export class Credentials {
     userPermission: Permissions | undefined,
     demandedPermission: Permissions
   ) {
-    if (!userPermission)
-      throw new Error('Unauthorized access. User permission is not valid')
-    if (userPermission < demandedPermission)
-      throw new Error('Unauthorized access')
+    // if (!userPermission)
+    //   throw new Error('Unauthorized access. User permission is not valid')
+    // if (userPermission < demandedPermission)
+    //   throw new Error('Unauthorized access')
```

Comment:
Weź to odkomentuj. I podeślij jeszcze raz jakiego errora to daje, bo trzeba by to w końcu ogarnąć xD

Reply:
zrobione
![image](https://user-images.githubusercontent.com/84126086/168814119-4992e30b-5043-4885-9ec6-31136f1d0078.png)